### ISSUE TITLE
Production ready?

### ISSUE THREAD

**Issue by westcoastavgallery** [KiteSync/realm-sync-js#153]
Just wondering if this is production ready? last commit was a while ago .
Just curious, im looking for something similar or rolling one my self thats why im asking.

**Comment by mikejonas** [comment]
Hi @westcoastavgallery. After speaking directly with realm and getting a sneak peak at what they were working on for an offline first syncing solution, we decided to deprecate this product. Check out what they built - https://realm.io/products/realm-mobile-platform/.  I think it's really awesome!!!

**Comment by simpleshadow** [comment]
@mikejonas Realm is awesome and their Realm Mobile Platform seems great, but KiteSync seemed like a great solution that would allow many to host their own backend and then build on top of it with our own or other third-party services... Now we have to pay to use Realm's platform (kudos to them for cornering this market). It isn't clear from their docs how one would integrate things like file hosting, etc. 

Shame this project has been abandoned...

**Comment by mikejonas** [comment]
You're totally right @simpleshadow, it would be awesome to have a solution that's backend and database agnostic. That was our initial plan. I hope someone can fork this repository or idea and continue.

If you're interested in how syncing works, check out these resources:
- [Edam Sync](https://dev.evernote.com/media/pdf/edam-sync.pdf) - A high-level overview of the logic behind Evernote's syncing
- [Ensembles](http://samples.leanpub.com/ensembles-sample.pdf) - (downloads a pdf). This is specific to core data, but it contains high-level thoughts on syncing, as well as insights on how they developed their API.

**Comment by Dids** [comment]
No news on a potential fork or continuing this one?

In my case, I have a MySQL database that I need to sync with Realm clients, and so far haven't found any Realm-related solutions, since the free version of RMP doesn't support any kind of interaction server side, so no custom adapters or bridges.

I'd hate to have to resort to rolling my own sync (JSON -> PHP -> MySQL), when there's literally a better alternative right at my fingertips.

**Comment by vladaman** [comment]
+1 we'd like to see open sourced sync solution on AWS and DynamoDB for realm. Having commercial product as an option is unfortunate. We can't depend on third party due to history of discontinued products like Parse. AWS lambda is best combination. We'd love to see this project to continue. Until then we have our own custom solution instead. May not be perfect but works.
